src/CMakeLists.txt

```diff
@@ -740,6 +740,7 @@ install(FILES
   sample.ceph.conf
   DESTINATION ${CMAKE_INSTALL_DOCDIR})
 
+set(PYTHON_EXECUTABLE "${PYTHON_EXECUTABLE}" STRING)
```

Comment:
hi @IlsooByun, thanks for fixing this issue. i think a better fix would be 

1. s/PYTHON_EXECUTABLE/MGR_PYTHON_EXECUTABLE/ in https://github.com/ceph/ceph/blob/49addda5707f06b90746bee96b82c4d77a23951b/src/include/config-h.in.cmake#L319. as per https://cmake.org/cmake/help/v3.2/command/configure_file.html?highlight=cmakedefine#configure-file what variable quoted by `@` is does not matters. so the existing .in file is misleading, probably we can make it more consistent? in this case, it should represent `MGR_PYTHON_EXECUTABLE`.
1. s/PYTHON_EXECUTABLE/MGR_PYTHON_EXECUTABLE/ in https://github.com/ceph/ceph/blob/49addda5707f06b90746bee96b82c4d77a23951b/src/mgr/PyModuleRegistry.cc#L44 and https://github.com/ceph/ceph/blob/49addda5707f06b90746bee96b82c4d77a23951b/src/mgr/PyModuleRegistry.cc#L47 as well

what do you think?

Reply:
I agree with you. I tested in a similar fashion but chose this approach to reduce source code changes. I'll change as you said and push it again.

Reply:
@IlsooByun probably we don't think change with the rest of the changes in this PR? actually, i still cannot understand why it addresses the failure you described in the tracker ticket..

Reply:
**cmakedefine** checks if the variable is already set. In this case, PYTHON_EXECUTABLE is not set, **cmakedefine** generates '/* undef PYTHON_EXECUTABLE*/'. So I had to set any value to PYTHON_EXECUTABLE to compile the code. Considering the case where PYTHON_EXECUTABLE has already been used elsewhere, this variable was set to its own value.

Reply:
I'm sorry for the confusion. I'll remove that line.